README.md

```diff
@@ -0,0 +1,39 @@
+# bugify-olivebay
+bugify-olivebay created by GitHub Classroom
+
+`bugify` is a Go library that allows you to auto-generate a GitHub issue to report a bug using the GitHub API.
+
+## Using the Go library
+
+Import the library using:
+
+```
+github.com/olivebay/bugify-olivebay
+```
+
+## Creating a client:
+
+Create a new Client object by calling `bugify.NewClient()` with your API key and repository: 
```

Comment:
Once you think the design has stabilised sufficiently, update the README for these changes. (When you think it's ready for use, add a suitable semver tag to the repo, like `v0.1.0`.)